src/renderer/renderer_elements.ts

```diff
@@ -1147,16 +1159,74 @@ export class InterstateEdge extends Edge {
         );
         const oldFont = ctx.font;
         ctx.font = '8px sans-serif';
-        const labelMetrics = ctx.measureText(this.label());
+        let label = '';
+        if (this.attributes().assignments) {
+            for (const k of Object.keys(this.attributes().assignments))
+                label += k + ' 🡐 ' + this.attributes().assignments[k] + '\n';
```

Comment:
This feels like it may not be supported everywhere (encoding dependent). Is there a way of adding a `\u` code?

Reply:
I addressed it by adding proper multi-line support (i.e., printing individual labels for each line). Thank you for pointing this out - the method I chose is only supported by _very_ few browsers and does not follow the `fillText` canvas API specifications.